packages/components/CHANGELOG.md

```diff
@@ -25,6 +25,7 @@
 -   `InputControl`: Fix focus style size ([#54394](https://github.com/WordPress/gutenberg/pull/54394)).
 -   `BorderControl`: Use standard focus style on BorderControl ([#54429](https://github.com/WordPress/gutenberg/pull/54429)).
 -   `Color values`: Update borderFocus to ADMIN.theme ([#54425](https://github.com/WordPress/gutenberg/pull/54425)).
+- `Placeholder`: Improved DOM structure and screen reader announcements ([#45801](https://github.com/WordPress/gutenberg/pull/45801)).
```

Comment:
Nit: spacing

```suggestion
-   `Placeholder`: Improved DOM structure and screen reader announcements ([#45801](https://github.com/WordPress/gutenberg/pull/45801)).
```

Reply:
@ciampo Would be nice if you could figure out a way to automate this via some Markdown plugin or something. Not being able to see, the Prettier and JSDoc are really nice as I can just select fix in VS Code and it takes care of it.